central/postgres/pruning.go

```diff
@@ -14,6 +16,9 @@ const (
 	pruneClusterHealthStatusesStmt = `DELETE FROM cluster_health_statuses child WHERE NOT EXISTS
 		(SELECT 1 FROM clusters parent WHERE
 		child.Id = parent.Id)`
+
+	getAllOrphanedAlerts = `SELECT id from alerts WHERE lifecyclestage = 0 and state = 0 and time < NOW() - INTERVAL '30 MINUTES' and NOT EXISTS
```

Comment:
we should probably pass the orphan window into here so we only have to manage that in one place.

Reply:
👍